content/en/developers/_index.md

```diff
@@ -12,13 +12,49 @@ further_reading:
+| JMX Beans                    | JVM-based applications expose JMX metrics that Datadog can collect with the JMX integration. For example, the [JMX integration][11] is already used for official integrations like [Solr][12], [Tomcat][13], [Cassandra][14], etc. |
+| Logs                         | Most applications generate some type of log. Use Datadog's [Log Management][15] for base monitoring. |
+| Network traffic and SNMP     | Network devices, such as routers, switches, printers, emit SNMP metrics (Simple Network Management Protocol) which can be collected using [Network Device Monitoring][16]. Additionally, [Network Performance Monitoring][17] tracks all network traffic in and out of a host. |
+| OpenMetrics                  | The Agent includes [OpenMetrics and Prometheus checks][18] capable of scraping Prometheus endpoints. |
+| Processes                    | To collect resource usage metrics for specific running processes on any host (CPU, memory, I/O, etc.) use the [Processes integration][19]. To query across all your running processes for open-ended debugging use [Live Process Monitoring][20] (like htop without having to SSH).|
+| Windows Performance Counters | Windows services or applications that expose performance counters can be collected with the [PDH integration][21]. |
+
+## Creating your own solution
+
+Developers have several choices for sending unsupported data to Datadog.
+
+- [**DogStatsD**][22] is a metrics aggregation service that accepts custom metrics, events, and service checks.
+
+- [**Custom checks**][23] enable you to collect metrics from custom applications or systems. [Custom Agent checks][23] are suitable for many basic needs. For more advanced requirements like metrics preprocessing, you may choose to write an [OpenMetrics][24] check. You may also wish to submit a [service][25] check within a custom Agent check.
+
+- [**Integrations**][26] also enable you to collect metrics from custom applications or systems. Integrations are reusable. You can keep your integration private, or you can write a public integration and contribute to Datadog's [repository of community integrations][27] to be used by other developers.
+
+
+### Should I write a custom check or an integration?
+
+The primary difference between custom checks and integrations is that integrations are reusable components that can become part of the Datadog's ecosystem. They generally take more effort (time to develop) and are best suited for general use-cases such as application frameworks, open source projects, or commonly used software. For more niche scenarios, such as monitoring services that are not widely used outside your team or organization, writing a custom check may be the most efficient option. However, you may choose to write an integration instead of a custom check if your particular use-case requires you to publish and deploy your solution as a Python wheel (`.whl`).
```

Comment:
```suggestion
The primary difference between custom checks and integrations is that integrations are reusable components that can become part of the Datadog's ecosystem. They generally take more effort (time to develop) and are best suited for general use-cases such as application frameworks, open source projects, or commonly used software. For more niche scenarios, such as monitoring services that are not widely used outside your team or organization, writing a custom check may be the most efficient option. However, you may choose to write an integration instead of a custom check if your particular use-case requires you to publish and deploy your solution as a Python wheel (`.whl`). Custom checks also use custom metrics, which have a cost associated, whereas if an integration gets accepted into the Datadog ecosystem, it is no longer considered a custom metric. For more information about how this might impact cost, see our the [Datadog Pricing Page][29]. 
```